cobbler/api.py

```diff
@@ -396,6 +397,48 @@ def menus(self):
 
     # =======================================================================
 
+    def get_item_resolved_value(self, item_uuid: str, attribute: str):
+        """
+        This method helps non Python API consumers to retrieve the final data of a field with inheritance.
+
+        This does not help with network interfaces because they don't have a UUID at the moment and thus can't be
+        queried via their UUID.
+
+        :param item_uuid: The UUID of the item that should be retrieved.
+        :param attribute: The attribute that should be retrieved.
+        :raises ValueError: In case a value given was either malformed or the desired item did not exist.
+        :raises TypeError: In case the type of the method arguments do have the wrong type.
+        :raises AttributeError: In case the attribute specified is not available on the given item (type).
+        :returns: The attribute value. Since this might be of type NetworkInterface we cannot yet set this explicitly.
+        """
+        if not isinstance(item_uuid, str):
+            raise TypeError("item_uuid must be of type str!")
+
+        if not validate.validate_uuid(item_uuid):
+            raise ValueError("The given uuid did not have the correct format!")
+
+        if not isinstance(attribute, str):
+            raise TypeError("attribute must be of type str!")
+
+        if not attribute.isidentifier():
+            raise ValueError("The given attribute was not a valid Python identifier!")
```

Comment:
I don't really get this check, why is it needed? `str.isidentifier()` checks if a `str` is a in `keyword.kwlist`, i.e. a reserved Python keyword. How is that related to attributes of items to look up?